### ISSUE TITLE
TheliaFormEvent::getName() semantic change

### ISSUE THREAD

**Issue by bcabarrou** [thelia/thelia#1761]
`Thelia\Core\Event\TheliaFormEvent` extends `Symfony\Component\EventDispatcher\Event` to provide an event object for form-related events, and overrides the `getName()` method to return the **name of the form** in the event. This is a problem, since `Event::getName()` originally returns the **name of the event**.

Unfortunately, fixing it is now probably a BC break ?

**Comment by lovenunu** [comment]
Good catch. I think we have to break BC to fix it, so the fix would be 2.3 only.

I think renaming `getName` to `getFormName` will be enough :)
